lib/pwauth/okta/api.go

```diff
@@ -31,3 +50,19 @@ func (pa *PasswordAuthenticator) PasswordAuthenticate(username string,
 func (pa *PasswordAuthenticator) UpdateStorage(storage simplestorage.SimpleStore) error {
 	return nil
 }
+
+// VerifyOTP
+func (pa *PasswordAuthenticator) ValidateUserOTP(authUser string, otpValue int) (bool, error) {
+	return pa.validateUserOTP(authUser, otpValue)
+}
+
+// Initialize and verify Push
+func (pa *PasswordAuthenticator) ValidateUserPush(authUser string) (PushResponse, error) {
+	return pa.validateUserPush(authUser)
+}
+
+// SetAuthnURL. For testing only, update the internal authURL so that the backend can be tested
```

Comment:
Please unexport this method.

Reply:
This will be needed by later PR so that we can do testing of the handler that calls these methods.

Reply:
How about a new function that creates a testing *PasswordAuthenticator?